### DISCUSSION TITLE
Cover Embed not working

### DISCUSSION THREAD

**Discussion by super80e** [advplyr/audiobookshelf#1250] [Q&A]
I recently got my server set up, when i embed metadata all text metadata will embed but the cover does not change for any of my files.  I cannot get it to embed a cover even when I convert to a m4b file as well, in fact when it does an m4b it ill leave the cover blank.  Anyone seen this and have a direction i can look in?

**Comment by sly-fox-git** [comment]
I can confirm. I converted three books and none of them have cover art. ABS v2.2.12
